tests/test_data/settings_old

```diff
@@ -234,6 +234,17 @@ remove_old_puppet_certs_automatically: 0
 # the choice of DHCP management engine is in /etc/cobbler/modules.conf
 manage_dhcp: 0
 
+# set to 1 to enable DHCP IPv6 address configuration generation.
+# This currently only works with manager.isc DHCP module (isc dhcpd6 daemon)
+# See /etc/cobbler/modules.conf whether this isc module is chosen for dhcp
+# generation.
+enable_dhcpv6: 1
+
+# set to 1 to enable DHCP IPv4 address configuration generation.
+# This currently only works with manager.isc DHCP module
+# Other dhcp modules ignore this and above settings
+enable_dhcpv4: 1
```

Comment:
Please remove it here and add it to `settings.yaml`. Comments can be left since they are removed anyway when the settings file is written back. Please include them in the docs though.